articles/overview/what-is-quantum-computing.md

```diff
@@ -0,0 +1,54 @@
+---
+title: What is quantum computing?
+description: Learn what quantum computing is, and what a quantum computer can do
+author: natke
+ms.author: nakersha
+ms.date: 10/14/2019
+ms.topic: article
+uid: microsoft.quantum.overview.what
+---
+
+# What is quantum computing?
+
+Quantum computing is the application of quantum physics to the field of computer science.
+
+The theory of quantum physics posits that matter, at a quantum level is simultaneously in multiple states. And those many states interfere with each other like waves in a tide pool.
+
+Quantum computing stores information in quantum states of matter and uses quantum operations to compute on that information, by harnessing and learning to program quantum interference.
+
+Quantum computing might sound daunting, but with the right resources you can start building quantum applications today.
```

Comment:
Don't know why, but I am inspired and ready to go. Honestly, this sentence is 🔥 .